File: 2486-append-characters-to-string-to-make-subsequence/2486-append-characters-to-string-to-make-subsequence.cpp
class Solution {
public:
    int appendCharacters(string s, string t) {
        int i=0,j=0,ans=0;
        while(i<s.size()&&j<t.size()){
            if(s[i]==t[j]){j++;ans++;}
            i++;
        }
        return t.size()-ans;

    }
};

File: 2486-append-characters-to-string-to-make-subsequence/2486-append-characters-to-string-to-make-subsequence.java
public class Solution {
    public int appendCharacters(String s, String t) {
        int i = 0, j = 0, ans = 0;
        while (i < s.length() && j < t.length()) {
            if (s.charAt(i) == t.charAt(j)) {
                j++;
                ans++;
            }
            i++;
        }
        return t.length() - ans;
    }
}
